cli/src/main/resources/application.yml

```diff
@@ -18,9 +18,21 @@ device:
 
 spring:
   profiles: receiver
+
 hono:
   client:
     port: 15671
+
+address:
+  resolver:
+    cacheNegativeTimeToLive: 0 #discard failed DNS lookup results immediately
+    cacheMaxTimeToLive: 0 #support DNS based service resolution
+    rotateServers: true
```

Comment:
Are these properties really something, that a user need to configure?

Reply:
Similar to the Address Resolution Timeout, there are many more properties in Vert.x hostname resolver (QueryResolverOptions).  For example, from the Operation-experts point of view, parameters such as searchDomains, maxQueries and queryTimeout are useful to tune based on the environment.  Now those can be configured.  IMHO users now have more flexibility to configure the hostname resolver if needed. Else default values are used as defined in the profile.

Reply:
My point was, that if we changed the defaults in a way that it is good for the application, it might be good to set these defaults in code. So the user CAN change them in the Yaml but it does not look like he should do it. But after thinking about - it might be good to see theses changed values directly in the configuration to be aware what happens.

Reply:
I see so too, that it is better to set default values in spring application.yml rather in the code.